src/applications/static-pages/health-care-manage-benefits/view-test-lab-results-page/components/App/index.js

```diff
@@ -0,0 +1,358 @@
+        itemType="http://schema.org/Question"
+        data-template="paragraphs/q_a"
+        data-entity-id={3506}
+        data-analytics-faq-section
+        data-analytics-faq-text="How can this tool help me manage my health care?"
+      >
+        <h2 itemProp="name" id="how-can-this-tool-help-me-mana">
+          How can this tool help me manage my health care?
+        </h2>
+        <div
+          data-entity-id={3506}
+          itemProp="acceptedAnswer"
+          itemScope
+          itemType="http://schema.org/Answer"
+        >
+          <div itemProp="text">
+            <div
+              data-template="paragraphs/wysiwyg"
+              data-entity-id={3508}
+              className="processed-content"
+            >
+              <p>
+                With this tool, you&apos;ll be able to view and keep a record of
+                your VA lab and test results.
+              </p>
+              <p>
+                <strong>You can use the tool to:</strong>
+              </p>
+              <ul>
+                <li>
+                  View some of your VA lab and test results (like blood tests)
+                </li>
+                <li>Add results from non-VA health care providers and labs</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        itemScope
+        itemType="http://schema.org/Question"
+        data-template="paragraphs/q_a"
+        data-entity-id={3509}
+        data-analytics-faq-section
+        data-analytics-faq-text="Am I eligible to use this tool?"
+      >
+        <h2 itemProp="name" id="am-i-eligible-to-use-this-tool">
+          Am I eligible to use this tool?
+        </h2>
+        <div
+          data-entity-id={3509}
+          itemProp="acceptedAnswer"
+          itemScope
+          itemType="http://schema.org/Answer"
+        >
+          <div itemProp="text">
+            <div
+              data-template="paragraphs/wysiwyg"
+              data-entity-id={3510}
+              className="processed-content"
+            >
+              <p>
+                You can use this tool to view and track your VA lab and test
+                results if you meet all of the requirements listed below.
+              </p>
+              <p>
+                <strong>Both of these must be true. You’re:</strong>
+              </p>
+              <ul>
+                <li>
+                  Enrolled in VA health care, <strong>and</strong>
+                </li>
+                <li>Registered as a patient at a VA health facility</li>
+              </ul>
+              <p>
+                <a href="/health-care/how-to-apply/">
+                  Find out how to apply for VA health care
+                </a>
+              </p>
+              <p>
+                <strong>And you must have one of these free accounts:</strong>
+              </p>
+              <ul>
+                <li>
+                  A{' '}
+                  <a href="">
```

Comment:
Missing link

Reply:
https://www.myhealth.va.gov/mhv-portal-web/upgrading-your-my-healthevet-account-through-in-person-or-online-authentication

Reply:
Do not open in new tab (e.g. no `target="_blank"`) 🙂